src/lib/components/explorer/FacetSideBar.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+  import { Accordion, ProgressRadial } from '@skeletonlabs/skeleton';
+  import SearchStore from '$lib/stores/Search';
+  import { updateFacetsFromSearch } from '$lib/services/dictionary';
+  import type { DictionaryFacetResult } from '$lib/models/api/DictionaryResponses';
+  import { browser } from '$app/environment';
+  import ErrorAlert from '../ErrorAlert.svelte';
+  import type { Facet } from '$lib/models/Search';
+  import FacetCategory from './FacetCategory.svelte';
+  let { searchTerm, selectedFacets } = SearchStore;
+
+  let facetsPromise: Promise<DictionaryFacetResult[]>;
+  if (browser) {
+    searchTerm.subscribe(() => {
+      facetsPromise = updateFacetsFromSearch($searchTerm, $selectedFacets);
+    });
+    selectedFacets.subscribe(() => {
+      facetsPromise = updateFacetsFromSearch($searchTerm, $selectedFacets);
+    });
+  }
```

Comment:
If wonder, when do these get unsubscribed, if ever? Does it double subscribe when you navigate to a different page then back? Skimming the svelete tutorials (https://learn.svelte.dev/tutorial/auto-subscriptions) maybe it would be better to put it into an `onMount` method in the cleanup section (https://learn.svelte.dev/tutorial/onmount) or `onDestroy`.